echo/server/dembrane/database.py

```
@@ -28,7 +28,7 @@
     declarative_base,
 )
 from pgvector.sqlalchemy import Vector  # type: ignore
-from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.dialects.postgresql import UUID, JSONB
```

Ruff (0.8.2)

`sqlalchemy.dialects.postgresql.JSONB` imported but unused

Remove unused import: `sqlalchemy.dialects.postgresql.JSONB`

(F401)